Dockerfile

```diff
@@ -6,10 +6,6 @@ RUN apt-get update && \
     apt-get install -y python3-pip && \
     pip3 install --upgrade pip
 
-# Copy and install requirements first so this layer doesn't change when copy the local files below
-COPY requirements.txt .
-RUN pip3 install -r requirements.txt
-ADD . .
-WORKDIR /src
-ENTRYPOINT ["python3", "noisepy.py"]
+RUN pip3 install noisepy-seis
```

Comment:
Probably good to add an `ARG` at the very top and put in specific version here, that way when you're tagging the docker image, you know the exact version of the package that you're using? See https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact for docs.

Example:

```
ARG VERSION=0.5.4
...

RUN pip3 install noisepy-seis==${VERSION}
```

What do you think?

Reply:
Makes sense!